supabase/migrations/20250916130500_create_file_access_table.sql

```diff
@@ -0,0 +1,87 @@
+-- Create table to store per-file access grants
+
+-- Simple scope enum for access level
+do $$ begin
+  if not exists (select 1 from pg_type where typname = 'file_access_scope') then
+    create type file_access_scope as enum ('read_only','admin');
+  end if;
+end $$;
+
+create table if not exists "public"."file_access" (
+    "id" uuid not null default gen_random_uuid(),
+    "file_id" uuid not null,
+    "artist_account_id" uuid,
+    "user_id" uuid,
+    "organization_id" uuid,
+    "granted_by" uuid,
```

Comment:
why does `granted_by` matter?
- This feels like another excessive column.
- Please remove all excessive columns and only ship the minimum columns required to achieve the base functionality.